fhf-datafangst-client/src/containers/BenchmarkView/BenchmarkView.tsx

```diff
@@ -78,6 +79,21 @@ export const BenchmarkView: FC = () => {
 
   const followVessels = user?.following.map((id) => fiskeridirVessels[id]);
 
+  const updateFollowVessels = (vessel: Vessel, isFollowing?: number) => {
```

Comment:
This function gets drilled all the way down to its only caller FollowListItem.tsx. Maybe just move the function down there, everything is accessible in the redux state anyway